eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -406,7 +406,7 @@ function IsPythonPackageVersionPublished($pkgId, $pkgVersion) {
 # Retrieves the list of all tags that exist on the target repository
 function GetExistingTags($apiUrl) {
   try {
-    return (Invoke-WebRequest -Method "GET" -Uri "$apiUrl/git/refs/tags"  -MaximumRetryCount 3 -RetryIntervalSec 10) | % { $_.ref.Replace("refs/tags/", "") }
```

Comment:
How did this script work before if it was a hard error?

Also for a debugging perspective it might be best to capture the return of Invoke-RestMethod and then do the processing.

Reply:
It was changed to `Invoke-WebRequest` because `Invoke-WebRequest-WithHandling` would exit the entire script with an error code if a tag was not already present in the repo. Looks like I misread it and used `Invoke-WebRequest` instead of `Invoke-RestMethod` ... I did test that it'd work properly in the case of no existing tags https://dev.azure.com/azure-sdk/internal/_build/results?buildId=453064&view=results but clearly overlooked the obvious 99% case. 

Props to @scbedd for recommending on waiting until most of our releases had gone out before putting this change in.